### PR TITLE
feat(BackUpSeedModal) added enter key press handling

### LINKED ISSUE #8309: Hitting enter should submit form when confirming seedphrase
Look here:

Screenshot from 2022-11-17 14-48-26

I'm hitting <Enter> but nothing happens. I'd expect this to confirm and send me to the next step.

### PR DESCRIPTION
**Pull request by alexandraB99**
Closes #8309 

### What does the PR do
BackUpSeedModal: added enter key press handling

### Affected areas
BackUpSeedModal

### CODE REVIEW

File: ui/app/AppLayouts/Profile/popups/BackupSeedModal.qml [hunk 1]
```diff
@@ -136,6 +137,9 @@ StatusStackModal {
             titleText: qsTr("Confirm word #%1 of your seed phrase").arg(d.firstRandomNo + 1)
             wordRandomNumber: d.firstRandomNo
             wordAtRandomNumber: root.privacyStore.getMnemonicWordAtIndex(d.firstRandomNo)
+            onEnterPressed: {
+                nextButton.clicked();
```
Comment (alexjba) [inline ui/app/AppLayouts/Profile/popups/BackupSeedModal.qml:141]: I think we need to check first if `nextButton` is enabled

Comment (alexandraB99) [reply ui/app/AppLayouts/Profile/popups/BackupSeedModal.qml:141]: @alexjba you think `inputText.valid` is not enough? (see `BackupSeedStepBase.qml`)

**Review by caybro — APPROVED** [review]
LGTM

modulo what AlexJB mentioned

File: ui/app/AppLayouts/Profile/popups/BackupSeedModal.qml (hunk 1 above)
Comment (alexjba) [reply ui/app/AppLayouts/Profile/popups/BackupSeedModal.qml:141]: Moving the `inputText.valid` out of `BackupSeedStepBase.qml ` would make it obvious on what is the condition in which `clicked` is sent. But it's up to you. No strong opinion on it.

**Review by alexjba — APPROVED** [review]
LGTM! 👍

**Pull request merged**